jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/TextField.java

```diff
@@ -22,96 +19,157 @@
  **/
 public class TextField extends UIBaseElement<TextFieldAssert>
         implements HasLabel, HasPlaceholder, IsInput {
-    private String input = ".//input|.//textarea";
-    private String slot = ".v-input__slot";
-    private String message = ".v-messages__message";
-    private String counter = ".v-counter";
-    private String prefix = ".v-text-field__prefix";
-    private String suffix = ".v-text-field__suffix";
-    private String icon = ".v-icon";
-
+    private static final String INPUT = ".//input|.//textarea";
+    private static final String SLOT = ".v-input__slot";
+    private static final String MESSAGE = ".v-messages__message";
+    private static final String COUNTER = ".v-counter";
+    private static final String PREFIX = ".v-text-field__prefix";
+    private static final String SUFFIX = ".v-text-field__suffix";
+    private static final String ICON = ".v-icon";
+
+    /** Get the text input field element inside the TextField.
+     *
+     * @return text input field as {@link UIElement}
+     */
     @JDIAction("Get '{name}' text input field")
     public UIElement textInputField() {
-        return find(input);
+        return find(INPUT);
     }
 
+    /** Get the icon element inside the TextField.
+     *
+     * @return icon as {@link UIElement}
+     */
     @JDIAction("Get '{name}' icon")
     public UIElement icon() {
```

Comment:
It may be worth using not a UIElement, but a special Icon() element to provide narrower functionality for working with an element, and not all actions available for any element. This applies to all methods that return a particular element.

Reply:
It is done by keeping the interface UIElement as expland.